src/EditorFeatures/Core/GoToDefinition/GoToDefinitionItemService.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.FindUsages;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.FindUsages;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.Editor.GoToDefinition
+{
+    internal abstract class AbstractGoToDefinitionItemService : ForegroundThreadAffinitizedObject, IGoToSymbolService
```

Comment:
I don't understand why this is abstract and a language service when neither implementation does anything but export it.